src/test/cli.build.test.ts

```diff
@@ -27,6 +27,21 @@ describe('Dev Containers CLI', function () {
 
 	describe('Command build', () => {
 
+		it('should fail to push --image-name with --push true', async () => {
+			let success = false;
+			const testFolder = `${__dirname}/configs/example-push`;
```

Comment:
Can we use some existing dev config instead  from `configs` folder of creating a new one?

Reply:
using `configs/example`